packages/pigment-css-react/tests/theme/fixtures/themed-component.output.js

```diff
@@ -0,0 +1,50 @@
+import { styled as _styled3 } from '@pigment-css/react';
```

Comment:
This file might not have been formatted correctly and will later report issue in CI. Can you verify? After prettier, imports to the same module are consolidated into one.

Reply:
I run `test:update` but it remains the same.

Reply:
If you look at `styled.output.js`, it becomes

```js
import { styled as _styled, styled as _styled2, styled as _styled3 } from '@pigment-css/react';
```

Reply:
So I am not sure what is correct.